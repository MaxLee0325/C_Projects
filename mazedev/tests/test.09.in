Alpha
END
Alpha Sierra
Romeo Alpha
Quebec Zulu
END
Alpha
END
