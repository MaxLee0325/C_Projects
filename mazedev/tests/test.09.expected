Oops: Cannot find room Sierra.
Oops: Cannot find room Romeo.
Oops: Cannot find room Quebec.
Oops: Cannot find room Zulu.
Entering room Alpha.
Ended in room Alpha.
